src/App.js

```diff
@@ -1,11 +1,32 @@
+import { BrowserRouter as Router, Route, Link, Switch } from "react-router-dom";
 import "./App.css";
+import Home from './Component/Home';
+import Movies from './Component/Movies';
+import Navbar from "./Component/Navbar";
+import People from "./Component/People";
+import Locations from "./Component/Locations";
+
 
 function App() {
   return (
     <div className="app">
-      <main>
-        <h1>Hello, world!</h1>
-      </main>
+      <Router>
+        <nav>
+          <ul>
+            <li><Link to="/">Home</Link></li>
+            <li><Link to="/movies">Movies</Link></li>
+            <li><Link to="/people">People</Link></li>
+            <li><Link to="/locations">Locations</Link></li>
+          </ul>
+        </nav>
```

Comment:
```suggestion
     <Navbar />
```
Your Navbar component needs a couple adjustments, but for the most part looked good! You can use it here.